webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTableHelpers.js

```diff
@@ -36,6 +36,19 @@ const buildTableRow = (subscription, availableQuantities, updatedQuantity) => {
 const buildTableRowsFromGroup = (subscriptionGroup, availableQuantities, updatedQuantity) => {
   const { open, subscriptions } = subscriptionGroup;
 
+  if (subscriptionGroup.subscriptions.length > 1) {
+    const temp = JSON.parse(JSON.stringify(subscriptionGroup.subscriptions[0]));
```

Comment:
is the JSON parsing here necessary? I feel like the data should already be an object we can work with

Reply:
@jturel It was a read only object. Couldn't modify it. So I created a copy.